remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -116,4 +131,162 @@ public void testDeleteTagById_nonExistentTag()
 
         then(taggingServiceMock).shouldHaveNoInteractions();
     }
+
+    @Test
+    public void testCreateTags()
+    {
+        final List<String> tagNames = List.of("tag1", "99gat");
```

Comment:
It took me too long to work out that `gat` was `tag` backwards!

Reply:
Should I fix it? 😂

Reply:
No - this is great. I guess we're testing a tag name can start with a number.